test/functional/rpc_fundrawtransaction.py

```diff
@@ -41,6 +41,7 @@ def setup_network(self):
         connect_nodes(self.nodes[0], 3)
 
     def run_test(self):
+        self.log.info("Setup, generating 126 nodes and syncing...")
```

Comment:
Do you mean 126 blocks? Also, I think there are only 122. Perhaps best to just omit the number?

Reply:
Thanks @jnewbery! 4 nodes and 122 blocks. Replacing with "Connect nodes, set fees, generate blocks, and sync"